test/ngSanitize/sanitizeSpec.js

```diff
@@ -280,6 +280,29 @@ describe('HTML', function() {
     expect(doc).toEqual('<p><img src="x"></p>');
   }));
 
+  describe('Custom white-list support', function() {
+    beforeEach(module(function($sanitizeProvider) {
+      $sanitizeProvider.addValidElements({
+        inlineElements: ['button'],
+        voidElements: ['input']
+      });
+
+      $sanitizeProvider.addValidAttrs(['type']);
+    }));
+
+    it('should not ignore custom white-listed element', function() {
```

Comment:
"should not ignore" is not very straight forward. I would change it to something like "should allow".